### ISSUE TITLE
np.uint16 sufficient for intensities?

### ISSUE THREAD

**Issue by liquidcarbon** [MannLabs/alphatims#155]
Hello, do I understand correctly that you are putting intensities into `np.uint16` that only goes to 65535?  I believe it should be `np.uint32`...

https://github.com/MannLabs/alphatims/blob/12b9a137dae8a13a8febebd6d62dde69c3f3e9e6/alphatims/bruker.py#L503

Consider this dataset that you can get from UCSD MASSIVE:
```
import os
from ftplib import FTP
ftp_url = 'ftp://massive.ucsd.edu/MSV000084402/'

def start_ftp():
    sample_path = 'MSV000084402/raw/SRM1950_20min_88_01_6950.d'
    ftp = FTP('massive.ucsd.edu')
    ftp.login()
    ftp.cwd(sample_path)
    return ftp

ftp = start_ftp()

os.mkdir('lipidomics.d')
with open('analysis.tdf_bin', "wb") as f:
    ftp.retrbinary("RETR " + 'lipidomics.d/analysis.tdf_bin', f.write)

with open('analysis.tdf', "wb") as f:
    ftp.retrbinary("RETR " + 'lipidomics.d/analysis.tdf', f.write)

# pip install opentimspy opentims_bruker_bridge
import opentims_bruker_bridge
from opentimspy.opentims import OpenTIMS

TIMS_FOLDER = 'lipidomics.d'
D = OpenTIMS(TIMS_FOLDER)

frames = pd.DataFrame(D.frames)
ms1_frame_numbers = frames.query("(MsMsType == 0) & (ScanMode != 0)").Id.to_numpy()
print(f'{D}, {len(frames)} frames, {len(ms1_frame_numbers)} MS1 frames')

MIN_INTENSITY = 500
ms1 = (
    pd.DataFrame(D.query(frames=ms1_frame_numbers[:500]))
    .query(f'intensity > {MIN_INTENSITY}')
    .reset_index(drop=True)
)
ms1[ms1.intensity > 65535].shape
```
Returns 1470 rows.
There are definitely intensities in the hundreds of thousands - outside `np.uint16` range.

**Comment by liquidcarbon** [comment]
What are the best practices in your group on interpreting intensities / feature calling?  Do you rely on `analysis.tdf_bin` values or on Bruker SDK values?

Here's an illustration of why the plurality of the definition of _intensity_ bothers me.  Same NIST SRM 1950 lipidomics dataset as before.  I extracted "chromatomobilogram" (XICOM) of a `cholesterol + H+ - water` peak.  Same window of retention times and ion mobilities.  As you can see, raw data (left) contains several minor features that become a lot less prominent (almost disappear) on the right.  

![image](https://user-images.githubusercontent.com/47034358/140263799-c451b061-01b8-42f4-97e1-2f1bb3ecba22.png)

Code for the plot:
```
atims = alphatims.bruker.TimsTOF(TIMS_FOLDER)
otims = OpenTIMS(TIMS_FOLDER)

fig, (ax1, ax2) = plt.subplots(figsize=(24,8), ncols=2)
plot_params = dict(
    kind='scatter',
    marker='x',
    lw=1.5,
    s=15,
    alpha=0.8,
    cmap='turbo',
    sharex=False,
)
(
#     atims[4000:,300:800,0,157376:157378]
    atims[9250:9950,360:450,0,157376:157378]
    .sort_values('intensity_values')
    .plot(
        ax=ax1, x='frame_indices', y='scan_indices',
        xlim=(9250, 9950), ylim=(360, 450),
        c='intensity_values',
        **plot_params,
    )
)
(
    pd.DataFrame(otims.query(frames=np.arange(9251,9950,3), columns=('frame','scan','tof','intensity')))
    .query('tof > 157375 & tof < 157378')
    .sort_values('intensity')
    .plot(
        ax=ax2, x='frame', y='scan',
        xlim=(9250, 9950), ylim=(360, 450),
        c='intensity',
        **plot_params,
    )
)
ax1.set_title('AlphaTIMS (analysis.tdf_bin raw intensities)')
ax2.set_title('OpenTIMS (Bruker SDK)')
plt.suptitle('Comparing AlphaTIMS & OpenTIMS XICOMs for $[C_{27}H_{45}]^+$')
plt.show()
```

**Comment by swillems** [comment]
I just had a call with Bruker about this, and indeed the accumlation time can/should be used as a correction factor. I'll include it in the next release of AlphaTims. For now I hope it is not too difficult to correct for manually...

**Comment by swillems** [comment]
As a quick fix, this is now available on the development branch (install with e.g. `pip install "git+https://github.com/MannLabs/alphatims.git@develop#egg=alphatims[plotting]" --upgrade`. 

In brief: Whenever a pd.DataFrame is returned, it now includes a "corrected_intensity_values" column.

Note that the corrected intensity values annot be used for slicing and are not displayed in the GUI directly (but the are available in the table view if this is turned on)

**Comment by swillems** [comment]
Dear @liquidcarbon. This fix has now been merged into the master branch and has been released in version 0.3.1. If this issue is not resolved properly, feel free to reopen this issue.
